sw/device/lib/crypto/aes_gcm/aes_gcm.c

```diff
@@ -165,6 +167,23 @@ static inline size_t get_nblocks(size_t sz) {
   return out;
 }
 
+/**
+ * Implements the inc32() function from NIST SP800-38D.
+ *
+ * This operation theoretically increments the "rightmost" (last) 32 bits of
+ * the input modulo 2^32. However, NIST treats all bitvectors as big-endian and
+ * C integers are little-endian. In order to increment we first need to reverse
```

Comment:
Fair, I'll fix the wording there to be more clear that this is the platform, not the C language. It's explicit that the cryptolib only supports little-endian systems -- in fact, it assumes it will run specifically on Ibex except for unit tests.